seed/seed20.go

```diff
@@ -762,3 +762,25 @@ func (s *seed20) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed20) LoadAutoImportAssertions(commitTo func(*asserts.Batch) error) error {
+	if s.model.Grade() == asserts.ModelDangerous {
+		autoImportAssert := filepath.Join(s.systemDir, "auto-import.assert")
+		af, err := os.Open(autoImportAssert)
+		// ignore if assertion file does not exists
+		if os.IsNotExist(err) {
+			return err
+		}
+		if err != nil {
+			return err
+		}
+		defer af.Close()
+		batch := asserts.NewBatch(nil)
+		if _, err := batch.AddStream(af); err != nil {
+			return err
+		}
+		err = commitTo(batch)
+		return err
```

Comment:
this could just be `return commitTo...`now

Reply:
I have reworded the function to use reverse logic when checking for dangerous model and simplified this with it.

Reply:
Did you mean `return nil` here? If so, we might want a unit test that checks that this error cannot occur.

Reply:
My comment looks silly and wrong in the "Conversation" page, please switch to "Files changed" to read it in context :-)

Reply:
@mardy yes, this check does not make anymore sense since we are passing error to the called regardless. And for that filtering this specific error does not make sense anymore, so I have removed this extra check and only check `if err != nil`

Reply:
Did you forget to push? I still see the `IsNotExist(err)` check in the diff.